opm/simulators/linalg/ParallelOverlappingILU0.hpp

```diff
@@ -1023,7 +1122,7 @@ class ParallelOverlappingILU0
     //! \brief The relaxation factor to use.
     const field_type w_;
     const bool relaxation_;
-
+    size_type interiorSize_;
```

Comment:
I do not have a better name, but interiorSize_ for something that might be CpGrid::numCells() sounds wrong.

Reply:
Nevermind